mergify_engine/github_events.py

```diff
@@ -74,6 +74,7 @@ def get_ignore_reason(event_type, data):
         "check_suite",
         "check_run",
         "refresh",
+        "github_app_authorization",
```

Comment:
I don't think the engine is going to handle those events, so it's fine it returns "unexpected"

Reply:
Ok, i'm going to look elsewhere